.github/workflows/verible-ci.yml

```diff
@@ -177,6 +193,7 @@ jobs:
     needs: [ Check, Build ]
     runs-on: ubuntu-20.04
     name: 📦 Release
+    if: ${{github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/head/master')}}
```

Comment:
Could the `refs/head/master` instead of `refs/heads/master` be the culprit ? Looks like a typo...

Reply:
Yes, that could certainly be it 😄 https://github.com/google/verible/pull/698

Reply:
Oh, also I created #699 in parallel (with the skipping already moved to the build step)